src/modules/fancyzones/editor/FancyZonesEditor/Models/LayoutModel.cs

```diff
@@ -107,7 +144,7 @@ public bool IsSelected
                 if (_isSelected != value)
                 {
                     _isSelected = value;
-                    FirePropertyChanged("IsSelected");
+                    FirePropertyChanged(IsSelectedStr);

```

Comment:
Do not hardcode strings for property names, use `nameof(IsSelected)` instead (same for every property) or even better change FirePropertyChanged to:

```
private void FirePropertyChanged([CallerMemberName] string propertyName = null)
{
      PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
}
```
And everywhere in the properties you will only call `FirePropertyChanged()` (no parameter)

Reply:
Done